### ISSUE TITLE
Certain NPC responses do not reflect what the player character says

### ISSUE THREAD

**Issue by AmarinReyny** [CleverRaven/Cataclysm-DDA#51951]
### Describe the bug

It's not much of a big deal, but I happened to notice just now that some of what the NPCs say aren't appropriate for what the player character says to them. See the screenshot for an example. I don't know how much of the dialogue is affected.

### Steps To Reproduce

1. Talk to an NPC about something, probably their backstory or the like
2. Find out if their responses to the PC's words are appropriate

### Expected behavior

I expect that the NPC will give an appropriate response to what my character says to them.

### Screenshots

![Inappropriate response](https://user-images.githubusercontent.com/60671985/135146358-8880651f-cb2a-4f06-b574-1f3359085b35.png)

### Versions and configuration

OS: Windows
OS Version: Windows 8.1 Pro
Game Version: ccf5e15 [64-bit]
Graphics Version: Tiles
Game Language: System language []
Mods loaded: [
Dark Days Ahead [dda],
Disable NPC Needs [no_npc_food],
Bionic Professions [package_bionic_professions],
Aftershock [aftershock],
Blaze Industries [blazeindustries],
C.R.I.T Expansion Mod [crt_expansion],
Magiclysm [magiclysm],
XEAS [XEAS],
Gun Cotton [guncotton],
DinoMod [DinoMod],
Youkai Enemies [Youkai_Disco],
Mining Mod [Mining_Mod],
Mole Mutation [dig_a_tunna],
More Survivor Stuff [more_survivor_stuff],
Mythical Martial Arts [MMA],
More City Locations [cityside],
No Rail Stations [No_Rail_Stations],
Compatible Nimian Manufacture Guns [NM_wpns],
No Fungal Monsters [No_Fungi],
Fuji's Military Profession Pack [fuji_mpp],
Magiclysm No Class Limit [magiclysm_no_class_limit],
Mutation Changes [Mutation_Changes],
Portable Autodoc [portable_autodoc],
SpeedyDex [speedydex],
Stats Through Kills [stats_through_kills],
Stats Through Skills [StatsThroughSkills],
Useful Helicopters [useful_helicopters],
Arcana and Magic Items [Arcana],
return_the_nut [return_the_nut],
Hydroponics [hydroponics],
Cooler Versatility [coolervers],
Chris K's Knives [ChrisKsKnives],
Custom Spells [Custom_Spells],
Reproduction mod [perverts],
Custom Bionics [custom_bionics],
Magiclysm Runes in Arcana Recipes [Custom_ArcanaRune],
Deconstruction Plus [Deconstruction_plus],
Core Data Changes [core_data_changes],
Arcadia Mapgen [custom_arcadia_mapgen]

**Comment by hexagonrecursion** [comment]
tldr: you pressed the back button and returned to the page you came from.

Relevant JSON:

https://github.com/CleverRaven/Cataclysm-DDA/blob/258f9054bf08ae5e17513f6bdb0f69c7b7a39fab/data/json/npcs/Backgrounds/evacuee_5.json#L18-L40

To reproduce:
1. Spawn and mind control an NPC
2. m[u]tate the NPC: remove the survivor story and replace it with Survivor Evacuee 5
3. Talk to the NPC
4. "I just wanted to talk for a bit."
5. "Mind if we just chat for a bit about your history?"
6. "How did you survive the Cataclysm?" or another question from this list https://github.com/CleverRaven/Cataclysm-DDA/blob/f59c6753f238b314cf0ba4cb24af3c2e55fb0951/data/json/npcs/talk_tags.json#L1401-L1414
7. "But bees aren't usually aggressive…"
    > "Well, excuse me if I didn't stop to <swear> ask what kind of killer bugs they were."
8. "Sorry.  Could you tell me more about them?"
    > "Yeah, I'm sure you've seen them, they're everywhere.  Like something out of an old sci-fi movie.  Some of the others in the evac shelter got stung, it was no joke.  I didn't stick around to see what the lasting effect was though.  I'm not ashamed to admit I ran like a <swear> chicken."
9. "Anyway…" or another reply from this list https://github.com/CleverRaven/Cataclysm-DDA/blob/f59c6753f238b314cf0ba4cb24af3c2e55fb0951/data/json/npcs/talk_tags.json#L1427-L1439
    > "Well, excuse me if I didn't stop to <swear> ask what kind of killer bugs they were."


- OS: Linux
    - OS Version: LSB Version: :core-4.1-amd64:core-4.1-noarch; Distributor ID: Fedora; Description: Fedora release 34 (Thirty Four); Release: 34; Codename: ThirtyFour; 
- Game Version: 0.F-4465-gef6206f0f7 [64-bit]
- Graphics Version: Tiles
- Game Language: English [en]
- Mods loaded: [
    Dark Days Ahead [dda],
    Disable NPC Needs [no_npc_food],
    No Fungal Growth [no_fungal_growth],
    Bionic Professions [package_bionic_professions]
]

**Comment by wapcaplet** [comment]
Appears to be working as intended, if a little awkwardly.
